.github/workflows/kind-e2e-cosigned.yaml

```diff
@@ -112,6 +112,11 @@ jobs:
         kubectl rollout status --timeout 5m --namespace cosign-system deployments/webhook
         kubectl wait deployment -n cosign-system --for condition=Available=True --timeout=90s --all
 
+    - name: Run Insecure Registry Tests
+      run: |
+        go install github.com/google/go-containerregistry/cmd/crane
```

Comment:
maybe we can use `  - uses: imjasonh/setup-crane@v0.2` to setup crane

Reply:
+1